.github/workflows/gitlab.yml

```diff
@@ -31,4 +31,4 @@ jobs:
           project-id: ${{ secrets.PROJECT_ID }}
           schedule: ${{ github.event_name == 'schedule' }}
           cancel-outdated-pipelines: ${{ github.ref_name != 'main' }}
-          triggered-ref: v2.4.3 # REMEMBER to also update in .gitlab-ci.yml
+          triggered-ref: LLT-5468_speedup_dev_builds # REMEMBER to also update in .gitlab-ci.yml
```

Comment:
Is this change temporary ?

Reply:
yes is part of test branch commit https://github.com/NordSecurity/libtelio/pull/823/commits/24feffc72391763b385318c57705ef11b7fbf7f0